src/ng-annotate-main.js

```diff
@@ -63,13 +63,30 @@ function matchMaterialShowModalOpen(node) {
     return false;
 }
 
+function getObjectExpressionReturnProperties(node) {
+    // matches object return via `return`:
+    // 1. function() { return {} }
+    // 2. () => { return {} }
+    if (node.type === "ReturnStatement" &&
+        node.argument && node.argument.type === "ObjectExpression") {
+        return node.argument.properties;
+    }
+
+    // matches object return via arrow function shortcut:
+    // 1. () => ({})
+    if (node.type === "ArrowFunctionExpression" && node.expression === true &&
+        node.body && node.body.type === "ObjectExpression") {
```

Comment:
The `node.body` check here is unnecessary - it cannot be null/undefined.

Reply:
done